### PR TITLE
Creating a Welcome page

### PR DESCRIPTION
**Pull request by Infinical**

### CODE REVIEW

**Review by LindaKadz — DISMISSED** [review]
Remove the user stories if you are not using them on this branch

**Comment by Infinical** [comment]
Screenshot from 2019-03-15 12-51-29
Screenshot from 2019-03-15 12-51-59

**Comment by Infinical** [comment]
Screenshot from 2019-03-15 12-51-29
Screenshot from 2019-03-15 12-51-59

**Comment by Infinical** [comment]
Screenshot from 2019-03-15 12-51-29

**Comment by Infinical** [comment]
Screenshot from 2019-03-15 12-51-59

File: features/list_books.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: List books on the welcome page
+
```
Comment (sigu) [inline features/list_books.feature]: Remove this space, the description is part of the feature

File: test/controllers/welcome_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```
Comment (sigu) [inline test/controllers/welcome_controller_test.rb]: We do not need this controller tests. Controller are covered by cucumber tests ( Feature tests)


This does not mean that controller tests are not important, it depends on the organisation. Some prefer and insist of these tests but as long as we can cover as much with `end to end tests` then IMO we do not need it

**Review by sigu — CHANGES_REQUESTED** [review]

**Review by sigu — DISMISSED** [review]

**Review by sigu — APPROVED** [review]

**Pull request merged**